locales/es.json

```diff
@@ -42,7 +42,6 @@
         "header": {
```

Comment:
- [ ] Check https://github.com/ourjapanlife/findadoc-localization/commit/f9b1baa2ff7450f07951fa613b74c8182fdd131c

Reply:
Since this file was first made outside lokalize, it wasn’t clobbered